deploy/kubernetes/fluentd-sumologic.yaml

```diff
@@ -76,10 +84,67 @@ data:
       @type datapoint
       tag "prometheus.datapoint"
     </match>
-    <filter prometheus.datapoint>
-      @type carbon_v2
-    </filter>    
     <match prometheus.datapoint>
+      @type rewrite_tag_filter
```

Comment:
What does this part do?

Reply:
The `rewrite_tag_filter` output plugin allows you to rewrite the tag for rerouting. In this case the plugin looks for the job and appends it to the already existing tag, which becomes `prometheus.datapoint.<JOB>`